From 85eef3e6db6f5c646631d11de0b834a13e81a971 Mon Sep 17 00:00:00 2001
From: Ran Wang <ran.wang_1@nxp.com>
Date: Thu, 9 Aug 2018 11:41:18 +0800
Subject: [PATCH 562/767] armv7:ls1021a: Correct addr of RCPM_IPPDEXPCR1

Otherwise Flex timer will not be able to wakeup system.

Signed-off-by: Ran Wang <ran.wang_1@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.09.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/arm/boot/dts/ls1021a.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/ls1021a.dtsi b/arch/arm/boot/dts/ls1021a.dtsi
index a58573f67e31..2faf1ffb3f6f 100644
--- a/arch/arm/boot/dts/ls1021a.dtsi
+++ b/arch/arm/boot/dts/ls1021a.dtsi
@@ -539,7 +539,7 @@
 		ftm0: ftm0@29d0000 {
 			compatible = "fsl,ls1021a-ftm-alarm";
 			reg = <0x0 0x29d0000 0x0 0x10000>,
-			      <0x0 0x1ee2140 0x0 0x4>;
+			      <0x0 0x1ee2144 0x0 0x4>;
 			reg-names = "ftm", "pmctrl";
 			interrupts = <GIC_SPI 118 IRQ_TYPE_LEVEL_HIGH>;
 			big-endian;
-- 
2.17.0

